src/pipeline/base_pipeline.py

```diff
@@ -46,10 +46,17 @@ def __init__(
         if args.pretrained is not None:
             self._load_pretrained(args.pretrained)
 
+        self.worker_outputs = {}
```

Comment:
Why do we need `worker_outputs` to be an attribute?

Reply:
It has the potential to be passed to the following workers